docs/decision_record.md

```diff
@@ -17,3 +17,22 @@ Should a wider GA4GH standard appear from [TASC issue 5](https://github.com/ga4g
 ### Known limitations
 
 - There have been a handful of reports of old sequences with disallowed characters in the sequence name rows (`>`) of FASTA files, particularly from the microbiome community. These sequence collections would have to be changed to include only SAM-compatible ASCII characters, which could restrict usage of the sequence collections protocols and delay uptake.
+
+## 2021-09-22 - Controlled vocabulary for the supported fields
+
+### Decision
+
+The attributes allowed to be used in the sequence collection digest construction are: 
+ - lengths: the length of the sequences
+ - sequences: the Refget digests of the sequences
+ - names: the given name of the sequence in this collection 
+ - topologies: the topology of each sequence (can be either linear or circular)
+ - priorities: importance of the sequence in the assembly
+
+### Linked issues
+
+- https://github.com/ga4gh/seqcol-spec/issues/8
+
+### Known limitations
+
+- After long discussion, we decided that storing an order array to explicitely was going to bring too many complications so we decided against it. This will limit (but not negate) our ability to find collections that have the same content but where the element. are in a different.
```

Comment:
I think I would just leave this for the other ADR entry that discusses it in detail.